cmd/main.go

```diff
@@ -222,22 +216,37 @@ func main() {
 		return pvcNames
 	}); err != nil {
 		setupLog.Error(err, "unable to set up field indexer")
-		os.Exit(1)
-	}
+		if err = (&scheduledtaskcontrollers.ScheduledTaskReconciler{
```

Comment:
issue: If err != nil, this should call os.Exit(1), now it after receiving an exit worthy error makes new reconcilers?

Reply:
I feel like there's a missing } here and that messed up the logic entirely.

Reply:
I think you're right, this is a bit of a mess. I've just killed off my changes and tried again, maybe this was why the e2e tests were failing on me. Good catch, thank you...